views/teamPage.ejs

```diff
@@ -4,11 +4,11 @@
     <br>
     <div class="text-center">
       <h1 class="display-2"><%= team[0].team %></h1>
-      <img height="200px" width="200px" src="<%= team[0].teamLogo %>">
+      <img height="200px" width="200px" id="teamLogo" src="<%= team[0].teamLogo %>">
```

Comment:
I know you did not initially add the height and width attributes, but can you remove the inline styles for the height and width and move it to the css file since it now has an id attribute.